di-ipv-evcs-stub/lambdas/src/services/evcsService.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  DynamoDB,
+  QueryInput,
+  PutItemInput,
+  UpdateItemInput,
+} from "@aws-sdk/client-dynamodb";
+import { marshall, unmarshall } from "@aws-sdk/util-dynamodb";
+
+import PostRequest from "../domain/postRequest";
+import ServiceResponse from "../domain/serviceResponse";
+import VcState from "../domain/enums/vcState";
+import EvcsVcItem from "../model/evcsVcItem";
+
+import { config } from "../common/config";
+import { getSsmParameter } from "../common/ssmParameter";
+import { v4 as uuid } from "uuid";
+import PatchRequest from "../domain/patchRequest";
+
+const dynamoClient = config.isLocalDev
+  ? new DynamoDB({
+      endpoint: config.localDynamoDbEndpoint,
+      region: config.region,
+    })
+  : new DynamoDB({ region: config.region });
+
+export async function processPostUserVCsRequest(
+  userId: string,
+  postRequest: PostRequest,
+): Promise<ServiceResponse> {
+  console.info(`Post user record.`);
+  for (const persistVC of postRequest.persistVCs) {
+    const vcItem: EvcsVcItem = {
+      userId: userId,
+      vc: persistVC.vc,
+      vcSignature: persistVC.vc.split(".")[2],
+      state: persistVC.state,
+      metadata: persistVC.metadata!,
+      provenance: persistVC.provenance!,
+      ttl: await getTtl(),
+    };
+    await saveUserVC(vcItem);
```

Comment:
Might be worth making this a [Promise All syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) so that we are processing all at the same time, instead of doing one-by-one for performance reasons

Reply:
done